src/types/entities/attachment.rs

```diff
@@ -2,7 +2,7 @@ use serde::{Deserialize, Serialize};
 
 use crate::types::utils::Snowflake;
 
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
```

Comment:
It's complicated. Let's say we have a random struct `MyStruct` which looks like this:
```rs
#[derive(PartialEq)]
struct MyStruct {
    something: Snowflake,
    else: AuditLogChange
    audit: AuditLog
}

struct AuditLog {
    users: Arc<Mutex<PublicUser>>
}
```
Then `MyStruct` cannot have `PartialEq` derived anymore, because somewhere down the line, there is a `Mutex<T>`, which can't have a PartialEq derive. Therefore, even though `MyStruct` wasn't changed but `AuditLog` was, `MyStruct` cannot have a PartialEq derive anymore. `PartialEq` can still be implemented manually, though.